src/pykeen/datasets/ogb.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+
+"""Load the OGB datasets.
+
+Run with python -m pykeen.datasets.ogb
+"""
+
+from typing import ClassVar, Optional
+
+import numpy as np
+
+from .base import LazyDataSet
+from ..triples import TriplesFactory
+
+__all__ = [
+    'OGBLoader',
+    'OGBBioKG',
+    'OGBWikiKG',
+]
+
+
+class OGBLoader(LazyDataSet):
+    """Load from the Open Graph Benchmark (OGB)."""
+
+    #: The name of the dataset to download
+    name: ClassVar[str]
+
+    def __init__(self, cache_root: Optional[str] = None, create_inverse_triples: bool = False):
+        self.cache_root = self._help_cache(cache_root)
```

Comment:
no `super` neccessary?

Reply:
correct. none of the base classes implement an `__init__()` so there's nothing to call